From 97adea3d387d269ff76dd8905e6f9b00e2f639ad Mon Sep 17 00:00:00 2001
From: Nikhil Badola <nikhil.badola@freescale.com>
Date: Tue, 9 Dec 2014 18:51:34 +0530
Subject: [PATCH 097/124] drivers: usb: Introduce FSL_USB2_PHY_UTMI_DUAL for
 usb gadget

Introduce FSL_USB2_PHY_UTMI_DUAL in gadget driver for setting
phy in SOCs with utmi dual phy

Signed-off-by: Nikhil Badola <nikhil.badola@freescale.com>
Change-Id: I2c53b89d9916bd17b5be8b5d9e32454943172d55
Reviewed-on: http://git.am.freescale.net:8181/25528
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Suresh Gupta <suresh.gupta@freescale.com>
Reviewed-by: Matthew Weigel <Matthew.Weigel@freescale.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index 3194aff..43c2841 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -260,6 +260,7 @@ static int dr_controller_setup(struct fsl_udc *udc)
 		portctrl |= PORTSCX_PTW_16BIT;
 		/* fall through */
 	case FSL_USB2_PHY_UTMI:
+	case FSL_USB2_PHY_UTMI_DUAL:
 		if (udc->pdata->have_sysif_regs) {
 			if (udc->pdata->controller_ver) {
 				/* controller version 1.6 or above */
-- 
2.1.0.27.g96db324

